docs/reference/ilm/getting-started-ilm.asciidoc

```diff
@@ -3,77 +3,87 @@
 [[getting-started-index-lifecycle-management]]
 == Getting started with {ilm}
 
-Let's jump into {ilm} ({ilm-init}) by working through a hands-on scenario.
-This section will leverage many new concepts unique to {ilm-init} that
-you may not be familiar with. The following sections will explore
-these in more details.
-
-The goal of this example is to set up a set of indices that will encapsulate
-the data from a time series data source. We can imagine there is a system
-like {filebeat-ref}[Filebeat] that continuously indexes documents into
-our writing index. We wish to roll over the index after it reaches a size
-of 50 gigabytes, or has been created 30 days ago, and then delete the index
-after 90 days.
+This tutorial demonstrates how to use {ilm} ({ilm-init}) 
+to manage indices that contain time-series data. 
+
+When you use a mechanism like Filebeat to continuously index documents into {es},
+you typically use an index alias so you can periodically roll over to a new index.
+This enables you to implement a hot-warm-cold architecture to meet your performance
+requirements for your newest data, control costs over time, enforce retention policies,
+and still get the most out of your data.
+
+To automate roll over and management of time-series indices with {ilm-init}, you:
+
+. <<ilm-gs-create-policy, Create a lifecycle policy>> with the {ilm-init} Put policy API.  
+. <<ilm-gs-apply-policy, Create an index template>> to apply the policy to each new time-series index.
+. <<ilm-gs-bootstrap, Bootstrap the initial time-series index>> by explicitly designating it as the write index.
+. <<ilm-gs-check-progress, Verify that indexes are moving through the lifecycle phases>> as expected with the 
+you use the {ilm-init} explain API. 
 
 [float]
 [[ilm-gs-create-policy]]
-=== Setting up a policy
+=== Create a lifecycle policy
 
-There are many new features introduced by {ilm-init}, but we will only focus on
-a few that are needed for our example. For starters, we will use the
-<<ilm-put-lifecycle,Put Policy>> API to define our first policy. Lifecycle
-policies are defined in JSON and include specific
-<<ilm-policy-definition,phases and actions>>.
+In your lifecycle policy, you specify the phases in your index lifecycle
```

Comment:
Shall we rephrase "In your lifecycle policy" to "When you create a lifecycle policy" ? ("in your ..." sounds a bit confusing as the user is just looking to create a policy now)